packages/rich-text/src/helpers/editor.ts

```diff
@@ -56,7 +71,11 @@ export function toggleBlock(editor, type: string): void {
   Transforms.setNodes(editor, newProperties);
 
   if (!isActive && (isList || isQuote)) {
-    const block = { type, children: [] };
+    const block = {
+      type,
+      data: {},
+      children: []
+    };
```

Comment:
This is one of many places in this PR where the compiler was sad because a `block` was missing a `data` attribute. I don't know what I did to make that a problem for this PR only, but if you see something similar added below that's probably why.